### PR TITLE
Fix development instructions

### PR DESCRIPTION
**Pull request by tiziano88**
A first pass at making the instructions work again after the Oak Classic code was removed.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -367,19 +237,19 @@ argument. For instance, the following command runs all fuzz targets with a 2
...
 ```bash
-./scripts/runner run-fuzz-targets -- -max_total_time=2
+runner run-cargo-fuzz -- -max_total_time=2
 ```
 
 The following lists all the `libFuzzer` options:
 
 ```bash
-./scripts/runner --logs run-fuzz-targets -- -help=1
+runner --logs run-fuzz-targets -- -help=1
```
Comment (rbehjati) [inline docs/development.md:246]:
```suggestion
runner --logs run-cargo-fuzz -- -help=1
```

File: docs/development.md [hunk 2]
```diff
@@ -367,19 +237,19 @@ argument. For instance, the following command runs all fuzz targets with a 2
...
 ```
 
 Moreover, `crate-name` alone or together with `target-name` could be specified
 to run all targets for a specific crate, or to run a specific target,
 respectively.
 
 ```bash
-./scripts/runner --logs run-fuzz-targets --crate-name=loader --target-name=wasm_invoke -- -max_total_time=20
+runner --logs run-fuzz-targets --crate-name=loader --target-name=wasm_invoke -- -max_total_time=20
```
Comment (rbehjati) [inline docs/development.md:254]:
```suggestion
runner --logs run-cargo-fuzz --crate-name=loader --target-name=wasm_invoke -- -max_total_time=20
```

**Review by rbehjati — APPROVED** [review]
Thanks :)

File: docs/development.md (hunk 1 above)
Comment (tiziano88) [reply docs/development.md]: Done.

File: docs/development.md (hunk 2 above)
Comment (tiziano88) [reply docs/development.md]: Done.

**Review by andrisaar — APPROVED** [review]

**Pull request merged**